static/js/popovers.js

```diff
@@ -68,8 +68,6 @@ function show_message_info_popover(element, id) {
         }
 
         var args = {
-            user_full_name: message.sender_full_name,
-            user_email: sender_email,
             user_id: message.sender_id,
             user_time: people.get_user_time(message.sender_id),
```

Comment:
You have pass user_time too to the `user_info_popover_title` template as the `user_info_popover_content` is no longer accessing  `user_time`